src/FSharp.Formatting.ApiDocs/GenerateModel.fs

```diff
@@ -1560,8 +1567,12 @@ module internal SymbolReader =
             let exampleNodes = doc.Elements(XName.Get "example") |> Seq.toList
             [ for (id, e) in List.indexed exampleNodes do
                 let html = new StringBuilder()
-                let n = if id = 0 then "example" else "example-" + string id
-                rawData.[n] <- e.Value
+                let exampleId =
+                    match e.TryAttr "id" with
```

Comment:
tied to the question in the [RFC change](https://github.com/fsharp/fslang-design/pull/630), what level of uniqueness should be enforced by tooling?

Reply:
Ideally fail if `--strict` is used, and warn otherwise

Reply:
I'm noodling around the implementation a bit here still for validation purposes: if the scope is just within a particular example then that's easy to do validation on, because we have all the examples locally, but at any other scope than that things get hard because we don't have that data available and/or we'd have to push the validation up a level...

Reply:
settled on duplicate within a member at the moment, but we could expand that to the type semi-easily if desired